Source/Controllers/Other/SecureBookmark.swift

```diff
@@ -8,28 +8,26 @@
 
 import Foundation
 
+final class SecureBookmark: NSObject {
+    private let theData: SecureBookmarkData
```

Comment:
Curious why you changed that from `Data` - is that internal property? If so, let's name it `bookmarkData`

Reply:
it's the first param to the ini fund